lib/repository/models/settings.dart

```diff
@@ -93,6 +93,12 @@ class Settings {
   final Rx<ThemeMode> theme = ThemeMode.system.obs;
   final Rx<SwipeDirection> fullscreenViewerSwipeDir = SwipeDirection.RIGHT.obs;
 
+  // Pin settings
+  final RxInt pinRowsPortrait = RxInt(3);
+  final RxInt pinColumnsPortrait = RxInt(3);
+  final RxInt pinRowsLandscape = RxInt(1);
+  final RxInt pinColumnsLandscape = RxInt(6);
```

Comment:
Seems like these settings aren't stored in the DB from what I see in the changes. I guess that isn't intended?

Reply:
It's intended for now. They're basically just placeholder variables so I don't hardcode values. When I make them changeable I'll add the configentry. Just don't want any broken settings being stored in the db until I'm sure I'm gonna use them. For example, right now the pinColumns in landscape are actually determined by device size, not the variable here.

So basically, until I properly support different device sizes, these are just dummy variables.